AoDDownloader/AoDDownloader.py

```diff
@@ -202,16 +202,16 @@ def set_playlist(self, anime_url: str, verbose: bool):
             self.current_playlist = [self._parse_episode(
                 episodeData) for episodeData in pd]
 
-    def download(self, verbose: bool, noBuffer: bool):
+    def download(self, verbose: bool, noBufferOutput: bool):
         for episode in self.playlist:
             if episode.exists:
                 click.echo(f"{click.style(f'Skipping {episode.title}.', fg='green')} Already exists.")
                 continue
             with tempfile.NamedTemporaryFile() if os.name != 'nt' else open(episode.file + ".ts", "w+b") as tmp:
                 with click.progressbar(episode.chunkList, label=f"Downloading {episode.title}:") as chunkList:
-                    for chunkUrl in chunkList:
-                        if noBuffer:
-                            click.echo("")
+                    for index, chunkUrl in enumerate(chunkList):
```

Comment:
I think what are you trying to do here could be done with clicks integrated progress bar option [`show_pos`](https://click.palletsprojects.com/en/7.x/api/#click.progressbar)

We should consider this option since the docs say:
> No printing must happen or the progress bar will be unintentionally destroyed.

Reply:
Most likely the "show_pos"-option won't work because its part of the not buffered progress bar.
I need output of the progress for the GUI that mustn't be buffered. Otherwise I won't be able to read it.

So far, everything works fine so I see no reason to change it.